Components/CustomDiv/CustomDiv.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { CSSProperties } from 'react';
+
+interface CustomeDivProps {
+  children: React.ReactNode;
+  style : React.CSSProperties;
+  width: number | string;
+  height: number | string;
```

Comment:
why number or string ?

Reply:
It should be string to accept % for example so I dropped the number
Done